front/pages/api/login.ts

```diff
@@ -280,4 +284,27 @@ async function handler(
   }
 }
 
+async function _createAndLogMembership(data: {
+  userId: number;
+  workspaceId: number;
+  role: "admin" | "user";
+  invitationFlow: "domain" | "email" | "personal";
+}) {
+  const m = await Membership.create({
+    role: data.role,
+    userId: data.userId,
+    workspaceId: data.workspaceId,
+  });
+
+  const tags = [
+    `workspace_id:${data.workspaceId}`,
+    `user_id:${data.userId}`,
+    `role:${data.role}`,
+    `invitation_flow:${data.invitationFlow}`,
+  ];
+  statsDClient.increment("user.membership_created", 1, tags);
```

Comment:
Could be `"workspace.membership_created"`?

Reply:
yeah workspace.membership_created is marginally better :+1: